index.html

```diff
@@ -586,11 +586,12 @@ <h3>
             reject |promise| with a "{{InvalidStateError}}" {{DOMException}},
             and run <a>abort the request steps</a>.
           </li>
-          <li>If |options|.|signal:AbortSignal|’s <a>aborted flag</a> is set, then
-          reject |promise| with an "{{AbortError}}" {{DOMException}},
-          and run <a>abort the request steps</a>.
+          <li>If |options|' |signal:AbortSignal| member is <a>present</a> and its
+          <a>aborted flag</a> is set, then reject |promise| with an "{{AbortError}}"
```

Comment:
Just a nit, split this into two sub-steps: 
1. reject. 
2. run abort.

Reply:
Done.